extensions/amp-compare-slider/0.1/amp-compare-slider.js

```diff
@@ -14,7 +14,11 @@
  * limitations under the License.
```

Comment:
Since it's a rename, and also `amp-compare-slider` is not launched, we should just delete these files.